src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -172,6 +172,8 @@ java.launcher.X.usage=\n\
 \                      (Linux Only) show host system or container\n\
 \                      configuration and continue\n\
 \    -Xss<size>        set java thread stack size\n\
+\                      The actual size may be rounded up to a multiple of the\n\
+\                      system page size as required by the operating system.\n\
```

Comment:
The Java manpage will also need this update.

Reply:
Thanks for reminder, I've update Java manpage at both locations.